app/models/concerns/auction_scopes.rb

```diff
@@ -3,6 +3,7 @@ module AuctionScopes
 
   included do
     scope :accepted, -> { where(status: statuses['accepted']) }
+    scope :accepted_pending_payment_url, -> { where(status: statuses['accepted_pending_payment_url']) }
     scope :delivery_accepted, -> { where(status: statuses['accepted']).where.not(accepted_at: nil) }
```

Comment:
you are right! updated at 261a2fb to remove all references of `delivery_accepted` and replace with `accepted` since now we know that all `accepted` auctions have a payment URL